public/locales/en/translation.json

```diff
@@ -48,7 +48,6 @@
   },
   "alerts": {
     "clearReset": "This will clear the whole canvas. Are you sure?",
-    "fileNotSelected": "A file was not selected.",
```

Comment:
Removed this since the code that used this message is no longer present in `master` (it was removed [here](https://github.com/excalidraw/excalidraw/pull/510#discussion_r369574941)). We may reintroduce it later.

Reply:
Let me also update these changes in the Spanish translation

Reply:
already done

Reply:
Sorry I didn't see that you did it 💯